### PR TITLE
Update text input border to grey 60 to match latest standards

### PR DESCRIPTION
**Pull request by cfarm**
As discussed on hub, updates to the checkbox input borders are slated. These updates include making the border grey-60. This PR makes text inputs grey-60 to match.

Pending pull requests dependent on this change:

- https://github.com/cfpb/capital-framework/pull/553
- https://github.com/cfpb/cfgov-refresh/pull/3209

screen shot 2017-08-07 at 1 32 07 pm

*Note the above screenshot also depicts the change from https://github.com/cfpb/capital-framework/pull/553 which makes the input height 27px with the padding at 10px 3px. The existing implementation had the wrong padding (7px all around), making the height taller than 27px.

### CODE REVIEW

**Comment by nataliafitzgerald** [comment]
@cfarm  - This looks perfect!

**Comment by nataliafitzgerald** [comment]
@cfarm 
Actually, now that I'm digging around I'm questioning the padding and height. I know the DM says "Padding: 10px 3px" but that DM page is so inconsistent and contradictory that I wouldn't want to make a change that we aren't certain about. The image you've shown above looks less tall than what we have now.

I took a look at the filter pattern and the height and padding look visually correct as they are:
screen shot 2017-08-07 at 4 42 33 pm

**Comment by nataliafitzgerald** [comment]
@cfarm 
Confirming that for the height and padding we will keep what we have now in terms of the code and update the specs to accurately reflect what's built. 

Border: 1 px #919395 (Grey 60)
Height: 35 px (includes border, padding, and text height)
Padding: 7px

**Review by nataliafitzgerald — APPROVED** [review]
The updated specs look correct.

**Review by ielerol — APPROVED** [review]
looks good to me

File: page-components/form-fields.md [hunk 1]
```diff
@@ -63,9 +63,9 @@ For screen readers, make sure the tab focus order is correct. Consider the order
 
 #### Default
 
-* Border: 1 px #75787b (Grey)
-* Height: 27 px
-* Padding: 10px 3px
+* Border: 1 px #919395 (Grey 60)
+* Height: 35 px (includes border, padding, and text height)
```
Comment (anselmbradford) [inline page-components/form-fields.md]: Is the space in `35 px` as you want it? (And `1 px` above it?)

File: page-components/form-fields.md [hunk 2]
```diff
@@ -63,9 +63,9 @@ For screen readers, make sure the tab focus order is correct. Consider the order
 
 #### Default
 
-* Border: 1 px #75787b (Grey)
-* Height: 27 px
-* Padding: 10px 3px
+* Border: 1 px #919395 (Grey 60)
```
Comment (anselmbradford) [inline page-components/form-fields.md]: If the variable is `gray-60`, etc. this should probably be `(Gray 60)`.

Comment (cfarm) [reply page-components/form-fields.md]: good catch.

File: page-components/form-fields.md (hunk 1 above)
Comment (cfarm) [reply page-components/form-fields.md]: The current implementation is 35px, yes (the screenshot in this PR is out of date, I removed those padding changes)

**Comment by cfarm** [comment]
@Scotchester @jimmynotjim can we get a merge?

File: page-components/form-fields.md (hunk 1 above)
Comment (Scotchester) [reply page-components/form-fields.md]: I think he's just talking about the space in "35 px" and "1 px", as opposed to "35px" or "1px" with no space.

File: page-components/form-fields.md (hunk 2 above)
Comment (Scotchester) [reply page-components/form-fields.md]: Are you going to push a fix for this?

File: page-components/form-fields.md (hunk 1 above)
Comment (anselmbradford) [reply page-components/form-fields.md]: yea what Scott said!

Comment (cfarm) [reply page-components/form-fields.md]: I SEE! will fix.

**Comment by cfarm** [comment]
@Scotchester pushed those changes. Do I need to commit the compiled CSS before merging?

**Comment by Scotchester** [comment]
> Do I need to commit the compiled CSS before merging?

Oh! Yes you do.

**Comment by cfarm** [comment]
Added compiled CSS!

**Comment by Scotchester** [comment]
:+1:

**Pull request merged**